README.md

```diff
@@ -1,33 +1,36 @@
-# intl_flavors
-Intl utilities to easily manage localization in Dart and Flutter
+# jimtl
+
+If you are tired on manually write ARB files or the limited Flutter localization support you are in the right place.
+
+With this package you'll be able to:
+
+- Generate ARB file from Dart code (done by build_runner)
+- Update your translations remotely (from external server or translation service)
+- Have flavors for your translations
 
 ## Setup
 
-Install last version of intl_flavors and intl_flavors_codegen.
+Install last version of jimtl for Dart or flutter_jimtl for Flutter and jimtl_codegen.
 
 ```yaml
 dependencies:
   intl:
-  intl_flavors:
+  flutter_jimtl:
 
 dev_dependencies:
   build_runner: ^2.0.3
-  intl_flavors_codegen:
+  jimtl_codegen:
 ```
 
-You then have two options, use Dart code to generate your ARB file, or manage your ARB file manually.
+## Your translation with the power of intl package
 
-We recommand you to manage your localization from Dart code, but choose the method you want and follow his setup.
+Intl is a great package to deal with translation, it let me write messages, plurals and genders in an easy and safe way.
```

Comment:
Intl is a great package to deal with translation, it allow you to write messages, plurals and genders in an easy and safe way.